src/analyses/variable-sensitivity/abstract_enviroment.cpp

```diff
@@ -517,3 +519,18 @@ std::vector<abstract_object_pointert> eval_operands(
 
   return operands;
 }
+
+class simplify_vsd_exprt : public simplify_exprt
```

Comment:
Having a custom simplification for VSD is not unreasonable but I am not immediately seeing what it adds in this case.

Reply:
It's gone!